core-service/src/main/java/com/crmfoodestablishment/coreservice/entity/Menu.java

```diff
@@ -0,0 +1,29 @@
+package com.crmfoodestablishment.coreservice.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class Menu {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false, unique = true)
+    private Integer id;
+
+    @Column(name = "name", nullable = false, length = 64)
+    private String name;
+
+    @Column(name = "comment", length = 128)
+    private String comment;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "season", nullable = false, length = 32)
+    private Season season;
+}
```

Comment:
We don`t need a list of Dishes here ? Class Dish include ManyToOne reference and fild Menu